packages/integration-karma/test/context/simple-context.spec.js

```diff
@@ -12,6 +12,45 @@ describe('Simple Custom Context Provider', () => {
         div.appendChild(elm);
         expect(elm.shadowRoot.textContent).toBe('ready');
     });
+
+    it('should call disconnect when no context value is provided', function () {
+        const div = document.createElement('div');
+        const elm = createElement('x-consumer', { is: Consumer });
+        document.body.appendChild(div);
+
+        const spy = {
+            connected: jasmine.createSpy('connected'),
+            disconnected: jasmine.createSpy('disconnected'),
+        };
+
+        installCustomContext(div, spy, true);
+        // not providing context intentionally
+        div.appendChild(elm);
+
+        expect(spy.connected.calls.count()).toBe(1);
```

Comment:
It's preferable to use `toHaveBeenCalledTimes()` instead of `toBe` here: https://jasmine.github.io/api/3.5/matchers.html#toHaveBeenCalledTimes.
If the test starts failing for any reason, it would make the error message easier to read.